reana/cli.py

```diff
@@ -157,7 +169,7 @@ def cli():  # noqa: D301
         $ cd reana-job-controller
         $ reana-dev git-checkout -b . 72 --fetch
         $ reana-dev docker-build -c .
-        $ kubectl delete pod -l app=job-controller
+        $ reana-dev kubectl-delete-pod -c reana-job-controller
```

Comment:
Can use `-c .` in the help here, so that it mirrors `docker-build -c .` command above.  Basically `.` means current working directory, so that people don't have to type much the component name....